yarn.lock

```diff
@@ -32131,6 +32131,11 @@ tempfile@^2.0.0:
     temp-dir "^1.0.0"
     uuid "^3.0.1"
 
+template-icons@^1.0.0:
```

Comment:
I believe that if you merge `master` into this branch, this change is gone from the PR.

Reply:
> Do you have an active SSE sandbox somewhere I can test with? I remember a conversation on Slack but can't find it at the moment :(

Here you go @tristandubbeld - https://veeexz-3000.preview.csb.app/s/clever-boyd-n41wst